tests/fuzz/config_validation_fuzzer.go

```diff
@@ -69,9 +69,43 @@ func FuzzConfigValidation(data []byte) int {
 	}
 
 	iobj = crdclient.TranslateObject(object, gvk, "cluster.local")
-	_, err = r.Resource().ValidateConfig(*iobj)
+	_, _ = r.Resource().ValidateConfig(*iobj)
+	return 1
+}
+
+func FuzzConfigValidation2(data []byte) int {
```

Comment:
Would be good to have a comment explaining why we have two. The PR description would be great

Reply:
Sure, let me just add that before it gets merged.